operator/src/main/resources/scripts/utils.sh

```diff
@@ -889,3 +889,94 @@ function adjustPath() {
     fi
   fi
 }
+
+#
+# checkCommonMount
+#   purpose: If the COMMON_MOUNT_PATH directory exists, it echoes the contents of output files
+#            in ${COMMON_MOUNT_PATH}/commonMountLogs dir. It returns 1 if a SEVERE message
+#            is found in any of the output files in ${COMMON_MOUNT_PATH}/commonMountLogs dirs.
+#            It also returns 1 if 'successfully' message is not found in the output files
+#            or if the COMMON_MOUNT_PATH directory is empty. Otherwise it returns 0 (success).
+#            See also 'commonMount.sh'.
+#
+function checkCommonMount() {
+  # check commonMount results (if any)
+  if [ -z "$COMMON_MOUNT_PATHS" ]; then
+    trace FINE "Common Mount: Skipping common mount checks (no common mounts configured)."
+    return
+  fi
+
+  trace FINE "Common Mount: COMMON_MOUNT_PATHS is '$COMMON_MOUNT_PATHS'."
+  for COMMON_MOUNT_PATH in ${COMMON_MOUNT_PATHS/,/ }; do
+    trace FINE "Common Mount: COMMON_MOUNT_PATH is '$COMMON_MOUNT_PATH'."
+    traceDirs $COMMON_MOUNT_PATH
+    touch ${COMMON_MOUNT_PATH}/testaccess.tmp
+    if [ $? -ne 0 ]; then
+      trace SEVERE "Common Mount: Cannot write to the COMMON_MOUNT_PATH '${COMMON_MOUNT_PATH}'. " \
+                   "This path is configurable using the domain resource 'spec.commonMountVolumes.mountPath' " \
+                   "attribute." && return 1
+    fi
+    rm -f ${COMMON_MOUNT_PATH}/testaccess.tmp || return 1
+
+    out_files=$(set -o pipefail ; ls -1 $COMMON_MOUNT_PATH/commonMountLogs/*.out 2>1 | sort --version-sort) \
```

Comment:
Minor: 
Add comment at 920: 
`# The container .out files embed their container name, the names will sort in the same order in which the containers ran`